src/l10n.js

```diff
@@ -1,4 +1,4 @@
-import { getGettextBuilder } from '@nextcloud/l10n/dist/gettext.js'
+import { getGettextBuilder } from '@nextcloud/l10n/gettext'
```

Comment:
@susnux @skjnldsv The documented way to import `getGettextBuilder` does not seem to work. 🤔